### ISSUE TITLE
Port to Godot 4

### ISSUE THREAD

**Issue by daf-lernen** [Wavesonics/LANServerBroadcast#11]
any chance of a godot 4 port? and if not how could this be done?

**Comment by mFieldHouses** [comment]
I have updated the plugin to be compatible with GD 4. I'm not sure how to upload it though, I made this account purely for this

**Comment by Wavesonics** [comment]
oh create a pull request and I can do it

**Comment by mFieldHouses** [comment]
Alright, I made a PR. I'm not sure whether it works though, because when I use it the serverListener doesn't see any servers being broadcast

**Comment by Zeuroux** [comment]
maybe server advertiser and listener have different port

**Comment by mFieldHouses** [comment]
No, it works. Turns out the listener and broadcaster just can’t find eachother when you have them running on the same device, but connecting between two devices works perfectly

**Comment by Zeuroux** [comment]
i tried and it works fine for me on same device and different :)

**Comment by mFieldHouses** [comment]
Ah, okay. Might just be a Linux thing then

**Comment by Wavesonics** [comment]
this has been ported to GD 4.5
